lib/plan_executor/app.rb

```diff
@@ -92,9 +92,10 @@ def validate_schema(schema, body)
       error = validate_schema(@schema, body)
       return [400, error.to_json] unless error.nil?
       name = body['plan_name']
-      # Errors if plan is not found
-      @pal.get_plan_info(name)
-
+      result = Concurrent::Future.execute(executor: @worker) do
```

Comment:
I think it's worth leaving the comment about why this is done -- otherwise looks fine to me

Reply:
Updated